packages/react-core/src/components/NotificationDrawer/NotificationDrawerHeaderCloseButton.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+
+interface NotificationDrawerCloseButtonProps extends React.HTMLProps<HTMLDivElement> {
+  /** Adds custom accessible text to the notification drawer close button. */
+  closeButtonAriaLabel?: string;
+  /** Optional icon other than x */
+  icon?: React.ReactNode;
+  /**  Callback for when close button is clicked */
+  onClose: () => void;
+}
+
+export const NotificationDrawerCloseButton: React.FunctionComponent<NotificationDrawerCloseButtonProps> = ({
+  closeButtonAriaLabel = 'Close',
+  icon,
+  onClose
+}: NotificationDrawerCloseButtonProps) => (
+  <div>
+    <button className="pf-c-button pf-m-plain" type="button" aria-label={closeButtonAriaLabel} onClick={onClose}>
```

Comment:
just curious, why wouldn't you use the react `<Button>` component?

Reply:
@mcoker i should, huh?